<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistRule</class>
 <widget class="QWidget" name="PlaylistRule">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="BuddyLabel" name="label_9">
       <property name="text">
        <string>Type:</string>
       </property>
       <property name="buddy">
        <cstring>typeCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeCombo">
       <item>
        <property name="text">
         <string>Include songs that match the following:</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Exclude songs that match the following:</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="BuddyLabel" name="label">
       <property name="text">
        <string>Artist:</string>
       </property>
       <property name="buddy">
        <cstring>artistText</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CompletionCombo" name="artistText">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="BuddyLabel" name="similarArtistsTextLabel">
       <property name="text">
        <string>Artists similar to:</string>
       </property>
       <property name="buddy">
        <cstring>similarArtistsText</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CompletionCombo" name="similarArtistsText"/>
     </item>
     <item row="3" column="0">
      <widget class="BuddyLabel" name="label_2">
       <property name="text">
        <string>Album Artist:</string>
       </property>
       <property name="buddy">
        <cstring>albumArtistText</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="CompletionCombo" name="albumArtistText"/>
     </item>
     <item row="4" column="0">
      <widget class="BuddyLabel" name="composerLabel">
       <property name="text">
        <string>Composer:</string>
       </property>
       <property name="buddy">
        <cstring>composerText</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="CompletionCombo" name="composerText">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="BuddyLabel" name="label_3">
       <property name="text">
        <string>Album:</string>
       </property>
       <property name="buddy">
        <cstring>albumText</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="CompletionCombo" name="albumText"/>
     </item>
     <item row="6" column="0">
      <widget class="BuddyLabel" name="label_4">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleText</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="LineEdit" name="titleText"/>
     </item>
     <item row="7" column="0">
      <widget class="BuddyLabel" name="label_5">
       <property name="text">
        <string>Genre</string>
       </property>
       <property name="buddy">
        <cstring>genreText</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="CompletionCombo" name="genreText"/>
     </item>
     <item row="8" column="0">
      <widget class="BuddyLabel" name="label_6">
       <property name="text">
        <string>From Year:</string>
       </property>
       <property name="buddy">
        <cstring>dateFromSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="dateFromSpin">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>0</height>
        </size>
       </property>
       <property name="specialValueText">
        <string>Any</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="BuddyLabel" name="label_6x">
       <property name="text">
        <string>To Year:</string>
       </property>
       <property name="buddy">
        <cstring>dateToSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="dateToSpin">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>0</height>
        </size>
       </property>
       <property name="specialValueText">
        <string>Any</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="BuddyLabel" name="commentLabel">
       <property name="text">
        <string>Comment:</string>
       </property>
       <property name="buddy">
        <cstring>commentText</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="LineEdit" name="commentText"/>
     </item>
     <item row="11" column="0">
      <widget class="BuddyLabel" name="filenameLabel">
       <property name="text">
        <string>Filename / path:</string>
       </property>
       <property name="buddy">
        <cstring>filenameText</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="LineEdit" name="filenameText"/>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QCheckBox" name="exactCheck">
       <property name="text">
        <string>Exact match</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3xx">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel"/>
   </item>
   <item>
    <widget class="NoteLabel" name="label_7">
     <property name="text">
      <string>Only enter values for the tags you wish to be search on.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NoteLabel" name="label_7y">
     <property name="text">
      <string>If 'Exact match' is checked, then string values will be matched for exact values. Otherwise partial matches will also be inclued. e.g. 'AB' would match 'ABBA'.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NoteLabel" name="label_7x">
     <property name="text">
      <string>For genre, end string with an asterisk to match various genres. e.g 'rock*' matches 'Hard Rock' and 'Rock and Roll'.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3xy">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>support/buddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>NoteLabel</class>
   <extends>QLabel</extends>
   <header>widgets/notelabel.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>support/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CompletionCombo</class>
   <extends>QComboBox</extends>
   <header>widgets/completioncombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
